e2e/benchmarks/local-benchmark/index.html

```diff
@@ -243,8 +221,9 @@ <h2>TensorFlow.js Model Benchmark</h2>
             console.warn(e.message);
           }
 
-          const debug = keepIntermediateTensors & (benchmarks[state.benchmark].supportDebug !== false);
-          referenceData = await predictAndGetPredictionData(predict, model, inferenceInput, debug);
+          const enableDump = keepIntermediateTensors & (benchmarks[state.benchmark].supportDump !== false);
```

Comment:
It seems that if `graphModel` is null, you even can't call `graphModel.getIntermediateTensors()`. In this case, the `enableDump` should also be false.
So ,how about changing it like below (
advantages: 1: we can know whether `enableDump` is supported at the beginning. 2: The `graphModel` can be reused in `predictAndGetData` and `dump`):
```
const graphModel = getGraphModel(model, benchmark);
const enableDump = keepIntermediateTensors & (benchmarks[state.benchmark].supportDump !== false) & graphModel !== null;

const referenceResult = await predictAndGetData(predict, model, inferenceInput, graphModel, enableDump);

...
if (enableDump) {
  ...
  await dump(graphModel, dumpInput, dumpPrefix, dumpLevel, dumpLength);
}
...

Reply:
As synced offline, keep getGraphModel private will be friendly to user.